router/client/src/client.rs

```diff
@@ -175,7 +175,7 @@ impl Client {
                     watermark: false,
                 }),
                 stopping_parameters: Some(StoppingCriteriaParameters {
-                    max_new_tokens: 10,
+                    max_new_tokens: std::cmp::min(10,max_total_tokens-max_input_length),
```

Comment:
Please, add spaces, no need to call std::cmp as it's defined in file's head (line 8).
Should be "cmp::min(10, max_total_tokens - max_input_length)"